playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss

```diff
@@ -215,7 +215,7 @@
       }
       .text_input {
         border-left: 0;
-        padding-left: 0;
+        padding-left: 16px;
```

Comment:
This is a potential breaking change, if it's going from 0 to 16px. We should make sure by checking nitro-web to see if people are using the specific prop that uses this scoped scss.

Reply:
I think I'll go ahead and approve based on this just being a small spacing push. We can update the instances in nitro-web accordingly after we release. It will only be small UI changes where devs have custom scss.